### PR TITLE
EAS-190: Uncomment xcom metrics

### PR DESCRIPTION
**Pull request by gonzalezzfelipe**

### CODE REVIEW

**Review by fpalladoro — APPROVED** [review]

**Review by Rcharriol — APPROVED** [review]

File: airflow_prometheus_exporter/xcom_config.py [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
 
-CONFIG_FILE = Path.cwd() / "config.yaml"
 
-
-def load_xcom_config():
+def load_xcom_config(path):
     """Loads the XCom config if present."""
     try:
-        with open(CONFIG_FILE) as file:
+        with open(path) as file:
```
Comment (tzulberti-jampp) [inline airflow_prometheus_exporter/xcom_config.py:7]: The path is the path to the config file or just the folder? If it is the config file, I would rename the varible to `config_path`

File: setup.py [hunk 2]
```diff
@@ -43,7 +43,7 @@
     packages=find_packages(include=['airflow_prometheus_exporter']),
     include_package_data=True,
     url='https://github.com/robinhood/airflow_prometheus_exporter',
-    version='1.1.2j',
+    version='1.1.3j',
```
Comment (tzulberti-jampp) [inline setup.py:46]: Check the __init__.py. There is a version variable there

Comment (tzulberti-jampp) [reply setup.py:46]: I am sorry, ignore this

**Comment by gonzalezzfelipe** [comment]
This wont be necessary, as Airflow will use the statsd client.

**Pull request closed without merging**